src/ts/components/subtitleoverlay.ts

```diff
@@ -36,7 +36,7 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     this.previewSubtitle = new SubtitleLabel({ text: 'example subtitle' });
 
     this.config = this.mergeConfig(config, {
-      cssClass: 'ui-subtitle-overlay',
+      cssClasses: ['ui-subtitle-overlay', 'default-subtitle'],
```

Comment:
Can we add positioning into the class name to make visible this is for subtitles which don't come with positioning attributes? `default-subtitle-positioning` or something.

Reply:
`subtitle-position-(default|left|top|...)`?